content/visualisation/visualisation_1.md

```diff
@@ -15,83 +15,62 @@ Run the command below to download the example dashboard:
 ```bash
```

Comment:
I can't comment above here, but I think you can remove the tabs from this section now, I think this will also fix the issues further down the page